PF/Assignment1/VehicleManagementOfException.java

```diff
@@ -0,0 +1,9 @@
+//  Exception class
+public class VehicleManagementOfException extends Exception {
+
+	private static final long serialVersionUID = 1L;
+
+	public VehicleManagementException(String message) {
```

Comment:
ACTION: As it should be constructor of this class so constructor name should be same as class name. Have you complied this before commit?